### PR TITLE
Space traveller's hub project

### PR DESCRIPTION
**Pull request by Elisha-Tetteyfio**
In this project, @tingamapuro04  and I designed a full space website with `missions` and `rockets` sections for checking out available missions or rockets by fetching data from the spacex API.
We also added user interactions for when you reserve a mission or make a reservation for a rocket, the interface changes to match your chosen actions. 
The `My Profile` section gets automatically updated based on interactions in the `Missions` and `Rockets` sections.

### CODE REVIEW

File: src/RocketSection/RocketList.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { useSelector } from 'react-redux';
+import Rocket from './Rocket';
+// import { getAllRocketsAsync } from '../reduxStore/rockets';
+
+function RocketList() {
+  // const dispatch = useDispatch();
+  const rocketTest = useSelector((state) => state.rocket);
+  // useEffect(() => {
+  //   dispatch(getAllRocketsAsync());
+  // }, [dispatch]);
```
Comment (omarsalem7) [inline src/RocketSection/RocketList.js]: - [x] Kindly remove the unused code to make your code clean and follow JavaScript best practices.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Elisha-Tetteyfio, @tingamapuro04 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Proper routes and view components created well ✔️
- Directories for all Redux state slice files created (rockets, missions) ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️

- [ ] Good use of Microverse readme template but you need to fill it with your data and update all sections in it.
- [ ]  _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors
- Good PR description

 ## Required Changes ♻️

- [ ] As asked by the previous reviewer, kindly update the Readme file of your project. Currently, it is too generic and doesn't give any information about the project. Consider updating all the sections with the relevant information about your project.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @Elisha-Tetteyfio, @tingamapuro04 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive Readme. 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Elisha-Tetteyfio** [comment]
At last.....thank you 😅

**Pull request merged**